paasta_tools/cli/cmds/validate.py

```diff
@@ -712,15 +713,21 @@ def validate_min_max_instances(service_path):
     return returncode
 
 
-def check_secrets_for_instance(instance_config_dict, soa_dir, service_path, vault_env):
+def check_secrets_for_instance(
+    instance_config_dict: InstanceConfigDict, soa_dir: str, service: str, vault_env: str
+):
```

Comment:
sorry, missed typing the retval!
```suggestion
) -> bool:
```